v2/spanner-common/src/test/java/com/google/cloud/teleport/v2/spanner/ddl/InformationSchemaScannerTest.java

```diff
@@ -355,16 +367,18 @@ public void testScanGSQLDdl() {
     Ddl ddl = informationSchemaScanner.scan();
     String expectedDdl =
         "CREATE TABLE `singer` (\n"
-            + "\t`singerId`                              STRING(50) NOT NULL,\n"
+            + "\t`singerId`                              INT64 NOT NULL,\n"
```

Comment:
why are we modifying the datatype of other columns?

Reply:
This is just to increase the coverage. The types were only testing STRING, added tests for INT64. Why INT64? Felt like SingerId would be an INT64 :)